test/integration-test.py

```diff
@@ -553,12 +553,23 @@ def run_chisel(test_case_filter):
         value()
 
 def run_loadtest():
-    # Run the load generator
-    latency_data_file = "/tmp/integration-test-latency.json"
+    """Run the load generator for v1 and v2."""
+    latency_data_file = "%s/integration-test-latency.json" % tempdir
     run("./bin/load-generator \
             -config test/load-generator/config/integration-test-config.json\
             -results %s" % latency_data_file)
 
+    latency_data_file = "%s/v2-integration-test-latency.json" % tempdir
+    subprocess.check_output(
+        "./bin/load-generator \
+            -config test/load-generator/config/v2-integration-test-config.json\
+            -results %s" % latency_data_file,
+        shell=True,
+        stderr=subprocess.STDOUT)
```

Comment:
Can you change this to use `run()` now that its available? I used `subprocess` directly for this in the V2 test because there was no `run` equiv.